components/mpas-seaice/src/shared/mpas_seaice_column.F

```diff
@@ -5557,6 +5647,310 @@ subroutine seaice_column_ocean_mixed_layer(domain)
 
   end subroutine seaice_column_ocean_mixed_layer
 
+!|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+!
+!  seaice_ocean_mixed_layer_e3sm
+!
+!> \brief E3SM ocean coupling proxy
+!> \author Adrian K. Turner, LANL
+!> \date 9th July 2022
+!> \details Representation of ocean processes and coupling in e3sm. This
+!>  routine mimics the ocean/sea ice coupling in e3sm using MPAS-Ocn,
+!>  including updates to ocean thickness, temperature and salinity from
+!>  interactions with sea ice and frazil formation.
```

Comment:
Is there a (design) document somewhere that explains these differences in true detail?  We will need that for the Icepack documentation.

Reply:
No design document. This is an additional subroutine in MPAS rather than the column package so doesn't necessarily need to be in icepack. Making this proxy was a good way to understand how frazil is currently done and I plan to try a better formulation so this may change soon anyway.